docs/user/aws/customization.md

```diff
@@ -20,6 +20,8 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
         This is only valid for `type` `io1`.
     * `size` (optional integer): Size of the root volume in gibibytes (GiB).
     * `type` (optional string):  The [type of volume][volume-type].
+    * `kmsKeyARN` (optional string): The [ARN of KMS key][ebs-kms-key] that should be used to encrypt the EBS volume.
+        Example ARN values are: `arn:aws:kms:us-east-1:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab` or `arn:aws:kms:us-east-1:111122223333:alias/my-key`
```

Comment:
mention the default KMS key here too?  Seem more important in this customer-ish facing docs than in the dev-only `variables-aws.tf` ;).

Reply:
fixed in 17030b3bdb997b8d5c64ec19b48e94da12f1be84